ts/src/bitget.ts

```diff
@@ -2654,8 +2655,14 @@ export default class bitget extends Exchange {
             }
             if (!isStopLossOrTakeProfit) {
                 request['size'] = this.amountToPrecision (symbol, amount);
+                const timeInForce = this.safeStringLower (params, 'timeInForce');
                 if (postOnly) {
                     request['timeInForceValue'] = 'post_only';
+                } else if (timeInForce === 'gtc') {
```

Comment:
@sc0Vu what if the user does not provide any timeINForce, will we set it as undefined?

Reply:
@carlosmiei yes you're right, updated.

Reply:
@sc0Vu we should unify this support for `timeInForce` as we do in the other exchanges, the user can pass either an exchange-specific timeInForce value like `ioc`, or the user can specify a unified value of timeInForce like `GTC` (all uppercase). Ideally that should be done by a lookup in params and then in the options. Please, check okx.ts or binance.ts for examples.

Reply:
@carlosmiei we already support post only for the spot order (only for plan order), I'll check others.

@kroitor I use lower string in case user type GtC or Gtc. The timeInForceValue is not required in bitget, should I add default value?

Reply:
@sc0Vu I would say that if the defaultValue is `normal` when not provided, we don't need to add it, but it does not hurt either